assets/dashboard.json

```diff
@@ -0,0 +1,700 @@
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": "YOUR_ACCOUNT_ID",
+                "query": "from Metric SELECT sum(prometheus_remote_storage_samples_retried_total) WHERE instrumentation.name = 'remote-write' FACET prometheus_server TIMESERIES AUTO LIMIT MAX"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "yAxisLeft": {
+              "zero": true
+            }
+          }
+        },
+        {
+          "title": "Failed Samples by Source",
+          "layout": {
+            "column": 10,
+            "row": 7,
+            "width": 3,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": "YOUR_ACCOUNT_ID",
+                "query": "FROM Metric SELECT rate(sum(prometheus_remote_storage_samples_failed_total), 1 second)  AS 'Samples Failed/Sec' FACET prometheus_server TIMESERIES AUTO LIMIT MAX"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "yAxisLeft": {
+              "zero": true
+            }
+          }
+        }
+      ]
+    },
+    {
+      "name": "Prometheus Server Monitoring",
+      "description": null,
+      "widgets": [
+        {
+          "title": "Memory by Source+Job",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "legend": {
+              "enabled": true
+            },
+            "nrqlQueries": [
+              {
+                "accountId": "YOUR_ACCOUNT_ID",
+                "query": "FROM Metric SELECT latest(process_resident_memory_bytes), latest(process_virtual_memory_bytes) WHERE instrumentation.name = 'remote-write' TIMESERIES AUTO FACET prometheus_server,job LIMIT MAX"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "yAxisLeft": {
+              "zero": true
+            }
+          }
+        },
+        {
+          "title": "CPU by Source+Job",
```

Comment:
why faceting by Job?

Reply:
I thought being more granular here It'd give more insights to the user about which job is consuming that CPU or memory... 🤔 Maybe that level of detail it's not relevant at all, WDYT?